components/cards/OurCrewCard.js

```diff
@@ -0,0 +1,16 @@
+import styles from "./OurCrewCard.module.css";
+
+export default function OurCrewCard({ image, position, name, description }) {
+    return (
+        <div className={styles.ourCrewItem}>
+            <div className={styles.ourCrewItemContainer}>
+                <img className={styles.ourCrewItemImage} src={image} alt={name} />
```

Comment:
This `alt="..."` text is not providing any extra info for the non-sighted users, so you can either change it or remove it.
(We are already having the crew members name in the `<h3>` below.)
- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA